apps/core/models.py

```diff
@@ -164,7 +164,7 @@ class Meta:
         )
 
     def __str__(self):
-        return self.reference
+        return str(self.reference)
```

Comment:
You'd better write a `__str__` method for the `data.models.Entry` model instead. Choose its representation carefully.